lib/class-wp-rest-swagger-controller.php

```diff
@@ -248,9 +248,16 @@ function ($matches) use (&$defaultidParams){
 						if(!empty($pdetails['type'])){
 							if($pdetails['type']=='array'){
 								$parameter['type']=$pdetails['type'];
-								$parameter['items']=array('type'=>'string');
+								$parameter['items']=array(
+									'type'=> $pdetails['items']['type'],
+									'properties'=> $pdetails['items']['properties']
```

Comment:
I don't think there are properties in an array type?

Reply:
Indeed, too much php for today.